packages/core/src/store/level-state-store.ts

```diff
@@ -1,17 +1,20 @@
-import Level from 'level-ts'
+import levelTs from 'level-ts'
+import type Level from 'level-ts'
 import { StreamState, StreamStateHolder, StreamUtils } from '@ceramicnetwork/common'
-import { StateStore } from './state-store'
-import StreamID from '@ceramicnetwork/streamid'
+import { StateStore } from './state-store.js'
+import { StreamID } from '@ceramicnetwork/streamid'
 import * as fs from 'fs'
 import path from 'path'
 
+const LevelC = (levelTs as any).default as unknown as typeof Level
```

Comment:
what does `LevelC` mean?  Can we give this a more descriptive name?

Reply:
Added comment there:

> When Node.js imports a CJS module from ESM, it considers whole contents of `module.exports` as ESM default export.
> 'level-ts' is a CommomJS module, which exports Level constructor as `exports.default`.
> This `default` name has no special meaning from ESM perspective. It is just yet another name.
> Types provided by level-ts though make it appear as ESM default.
> 
> So, here we untangle this mess, even if ugly.
> We import type information separately from code information, and then make sure we can access
> a properly typed constructor of `Level` (thus `LevelC`) exported from level-ts package.
>
> See also https://github.com/nodejs/node/blob/master/doc/api/esm.md#commonjs-namespaces

Reply:
Ah got it, that's helpful thanks!  Can we call it LevelConstructor then?  LevelC doesn't really convey any meaning to me

Reply:
Oh I see, you actually added all that as a comment to the code.  That's great and addresses my concern, thanks!